input/pagecontent/transport.md

```diff
@@ -1 +1,56 @@
-here is a sample.
+## GenomeX data transport
+
+Precision Medicine workflows require sharing complex data types with many relationships.
+Multiple transmission formats and protocols are used to transport Genomic test results from laboratories to health care organizations.
+HL7v2 transmission, using the MLLP protocol, is ubiquitously adopted as a result of healthcare law and remains the only guaranteed transport 

```

Comment:
Not sure if it is true about the phrase 'as a result of healthcare law'. Suggest to take it out.

Reply:
@kinsonho-arterys please review updated introduction and open new comments.  I will resolve these.